presto-docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -0,0 +1,368 @@
+specified in ``user`` attribute.
+
+For group names, a rule can be applied if at least one group name of this user
+matches the ``group`` regular expression.
+
+.. note::
+
+    By default, all users have access to the ``system`` catalog. You can
+    override this behavior by adding a rule.
+
+    Boolean ``true`` and ``false`` are also supported as legacy values for ``allow``,
+    to support backwards compatibility.  ``true`` maps to ``all``, and ``false`` maps to ``none``.
+
+For example, if you want to allow only the user ``admin`` to access the
+``mysql`` and the ``system`` catalog, allow users from the ``finance``
+and ``admin`` groups access to ``postgres`` catalog, allow all users to
+access the ``hive`` catalog, and deny all other access, you can use the
+following rules:
+
+.. code-block:: json
+
+    {
+      "catalogs": [
+        {
+          "user": "admin",
+          "catalog": "(mysql|system)",
+          "allow": "all"
+        },
+        {
+          "group": "finance|human_resources",
+          "catalog": "postgres",
+          "allow": true
+        },
+        {
+          "catalog": "hive",
+          "allow": "all"
+        },
+        {
+          "user": "alice",
+          "catalog": "postgresql",
+          "allow": "read-only"
+        },
+        {
+          "catalog": "system",
+          "allow": "none"
+        }
+      ]
+    }
+
+For group-based rules to match, users need to be assigned to groups by a
+:doc:`/develop/group-provider`.
+
+Schema Rules
+------------
+
+These rules allow you to grant ownership of a schema. Having ownership of an
+schema allows users to execute ``DROP SCHEMA``, ``ALTER SCHEMA`` (both renaming
+and setting authorization) and ``SHOW CREATE SCHEMA``. The user is granted
+ownership of a schema, based on the first matching rule read from top to
+bottom. If no rule matches, ownership is not granted. Each rule is composed of
+the following fields:
+
+* ``user`` (optional): regex to match against user name. Defaults to ``.*``.
+* ``group`` (optional): regex to match against group names. Defaults to ``.*``.
+* ``schema`` (optional): regex to match against schema name. Defaults to ``.*``.
+* ``owner`` (required): boolean indicating whether the user is to be considered
+  an owner of the schema. Defaults to ``false``.
+
+For example, to provide ownership of all schemas to user ``admin``, treat all
+users as owners of ``default`` schema and prevent user ``guest`` from ownership
+of any schema, you can use the following rules:
+
+.. code-block:: json
+
+    {
+      "catalogs": [
+        {
+          "allow": true
+        }
+      ],
+      "schemas": [
+        {
+          "user": "admin",
+          "schema": ".*",
+          "owner": true
+        },
+        {
+          "user": "guest",
+          "owner": false
+        },
+        {
+          "schema": "default",
+          "owner": true
+        }
+      ]
+    }
+
+Table Rules
```

Comment:
fyi @rosewms .. table rules are covered already. .. you might not have to do any more doc

Reply:
Those were added in https://github.com/prestosql/presto/pull/4668. This just moves the file to a separate page, since it seems nicer to read that way.

Reply:
Agreed.. there is just an issue with adding that doc and now we know we already got it..